FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Ayaz Rahman on 16/7/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal struct FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+
+		var feedImages: [FeedImage] {
+			return items.map { $0.feedImage }
+		}
+	}
+
+	private struct Item: Decodable {
+		let id: UUID
+		let description: String?
+		let location: String?
+		let imageURL: URL
+
+		enum CodingKeys: String, CodingKey {
+			case id = "image_id"
+			case description = "image_desc"
+			case location = "image_loc"
+			case imageURL = "image_url"
+		}
+
+		var feedImage: FeedImage {
+			return FeedImage(id: id, description: description, location: location, url: imageURL)
+		}
+	}
+
+	private static let OK_200: Int = 200
+
+	internal static func map(_ data: Data, from response: HTTPURLResponse) -> FeedLoader.Result {
```

Comment:
The convention in this codebase is to omit the default access leve keyword `internal`, please remove it.
```suggestion
	static func map(_ data: Data, from response: HTTPURLResponse) -> FeedLoader.Result {
```